lib/stitches/generator_files/app/controllers/api/api_controller.rb

```diff
@@ -1,10 +1,20 @@
 class Api::ApiController < ActionController::Base
   include Stitches::Deprecation
+  #
+  # The order of the rescue_from blocks is important - ActiveRecord::RecordNotFound must come after StandardError,
+  # otherwise "not_found" errors will get rescued in the StandardError block.
+  # See the documentation for rescue_from for further explanation:
+  # https://apidock.com/rails/ActiveSupport/Rescuable/ClassMethods/rescue_from
+  # Specifically, this part: "Handlers are inherited. They are searched from right to left, from bottom to top, and up
+  # the hierarchy."
+  #
+  rescue_from StandardError do |exception|
+    Bugsnag.notify(exception)
+    render json: { errors: Stitches::Errors.from_exception(exception) }, status: :internal_server_error
+  end
+
   rescue_from ActiveRecord::RecordNotFound do |exception|
-    respond_to do |type|
```

Comment:
I removed the `respond_to` block, because it's not needed. I learned from @davetron5000 that [Stitches only lets JSON requests through](https://github.com/stitchfix/item-price-service/pull/15#discussion_r213859374). Here is the [middleware](https://github.com/stitchfix/stitches/blob/fc49a8582c3b3ef4483f743fca2623862d4fbc78/lib/stitches/valid_mime_type.rb#L16) for references.

Reply:
Yup, makes sense.